src/app/(marketing)/page.tsx

```diff
@@ -31,7 +31,7 @@ export default async function IndexPage() {
             Dorf is a free, open source visual form builder for capturing
             feedback, leads, and opinions.
           </p>
-          <div className="space-x-4">
+          <div className="space-x-4 space-y-4">
```

Comment:
Thanks for contributing. Good catch! I think with these changes it still looks a bit off. I have a suggestion for this:
```suggestion
          <div className="flex flex-col md:flex-row gap-2">
```

Reply:
Ah yeah, definitely! 😅 I have a very slow computer, so I couldn’t set up the developer environment. I was depending on my luck hoping it'd work as expected. Great suggestion!